cmd/sonobuoy/app/operations/logs.go

```diff
@@ -17,11 +17,62 @@ limitations under the License.
 package operations
 
 import (
-	"errors"
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 )
 
-// Logs gathers the logs for the containers in the sonobuoy namespace and prints them out in a structured format
-func GetLogs( /*opts?*/ ) error {
-	// This should be straight forwards
-	return errors.New("not implemented")
+// Logs gathers the logs for the containers in the sonobuoy namespace and prints them
+
+// TODO(chuckha) should this be an option?
+const sonobuoyNamespace = "heptio-sonobuoy"
```

Comment:
It is pulled in from config, we should leverage the init routine to grab that data.